src/libre/dialect/pcre/parser.sid

```diff
@@ -149,6 +151,8 @@
 			class-literal: () -> (node :ast_expr) = {
 				{
 					(c, !, !) = ESC;
+				||
+					(c, !, !) = NOESC;
```

Comment:
:+1: 

We looked at this - it seems it was present when this grammar was first introduced, but then somehow got lost over time.